prow/plugins/golint/golint.go

```diff
@@ -76,8 +76,8 @@ type githubClient interface {
 
 const defaultConfidence = 0.8
 
-func minConfidence(g *plugins.Golint) float64 {
-	if g == nil || g.MinimumConfidence == nil {
+func minConfidence(g plugins.Golint) float64 {
+	if g.MinimumConfidence == nil {
```

Comment:
So yeah in this specific case, `nil` _is_ useful because it's  important to be able to tell the difference between "unset" and "set to `0.0`". In the former case we default to `0.8`, in the latter, the user specifically set it to `0.0`.